Client/src/Pages/Settings Pages/StudentSettings/ProfileSettings/StudentProfileSettings.js

```diff
@@ -261,16 +300,19 @@ const ProfileInformation = (props) => {
             className={classes.form}
             onChange={(e) => setBio(e.target.value)}
           />
-          <Button
+        <Fab
+            disabled={setDisabled()}
```

Comment:
just use 
`disabled={displaySaveButton()==0.5}`

Or if you prefer using a separate function, still you can just use return `displaySaveButton() == 0.5`